changelog/unreleased/chore-fix-vue-console-warnings.md

```diff
@@ -0,0 +1,6 @@
+chore: fix vue console warnings
```

Comment:
We're leaving out internal/dev changes from the changelog. Only end user facing changes should be covered by the changelog.

Reply:
Dropping it should also heal the CI